fix: FIT-997: YOLO export error in async_export

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/19772749322)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/683
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19772749322

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19829153466)
> Successfully merged:  create mode 100644 web/libs/storybook/public/logo.svg

matt-bernstein committed:
Merge branch 'develop' into 'fb-FIT-997'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19829153466

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19829848554)
> Successfully merged:  2 files changed, 2 insertions(+), 1 deletion(-)

matt-bernstein committed:
Merge branch 'develop' into 'fb-FIT-997'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19829848554

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/19831075307
